src/plone/restapi/interfaces.py

```diff
@@ -1,4 +1,9 @@
 # -*- coding: utf-8 -*-
+
```

Comment:
I usually disable those warnings globally because I don't think it is worth the effort. Though, that's just my personal opinion...